src/studio/src/designer/backend/Configuration/ConsoleTraceService.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Yuniql.Extensibility;
+
+namespace Altinn.Studio.Designer.Configuration
+{
+    /// <summary>
+    /// Copied from sample project.
+    /// </summary>
+    public class ConsoleTraceService : ITraceService
```

Comment:
maybe this could be excluded from code coverage

Reply:
Removed from code coverage